src/content/docs/guides/Utilities/0-useful-utilities.mdx

```diff
@@ -374,23 +374,23 @@ SplashKit's [Is Integer](/api/utilities/#is-integer) and [Is Number](/api/utilit
   */
   int read_integer(string prompt)
   {
-    string buffer;
+    string line;
 
     //Continue requesting user input until a valid whole number is entered.
     while (1)
     {
       //Prompt the user with the message
       write(prompt);
       //Read the user input as a string.
-      buffer = read_line();
+      line = read_line();
       //If the user input is a valid whole number, leave the loop.
-      if ( is_integer(buffer) )
+      if ( is_integer(line) )
         break;
       //If the user input was not a valid whole number, ask them to enter a whole number.
       write_line("Please enter a valid whole number.");
     }
     //Convert the user input to an integer before returning it.
-    return convert_to_integer(buffer);
+    return convert_to_integer(line);
   }
```

Comment:
Can you also swap this read_integer code (but keep comments where appropriate) for the one in programmers.guide on this page: https://programmers.guide/book/part-2-organised-code/2-organising-code/1-tour/04-00-utilities/#implementing-the-utilities (to remove the use of an infinite loop)? Thanks!

Reply:
I have written, tested, and commited this change.
![image](https://github.com/user-attachments/assets/7c2bdaef-7b59-4e8d-a8f6-dbbe547d2b32)